src/vs/workbench/browser/parts/editor/media/editortabscontrol.css

```diff
@@ -46,4 +46,10 @@
 	border-radius: 10px;
 	font-size: 12px;
 	position: absolute;
+	/* Browsers apply an effect to the drag image when the div becomes to large which makes them unreadable.
```

Comment:
Can you share an example how this was broken?

Reply:
![ShareX_s7GUTEhESh](https://github.com/microsoft/vscode/assets/44439583/f24c42e6-60ac-4fb2-8792-e1075c1005e9)

Reply:
Learned something.